src/middlewares/passport.js

```diff
@@ -0,0 +1,19 @@
+import { Strategy, ExtractJwt } from 'passport-jwt';
```

Comment:
I like the Idea of using passport to validate our tokens. But as we are not . using the tokens from social media, why do you use passport just to validate a token?

Reply:
I am not validating token, I'm decoding the token both while using social media or signup on the app by creating account